src/Symfony/Component/Validator/Tests/Constraints/EnumValidatorTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\Choice;
+use Symfony\Component\Validator\Constraints\Enum;
+use Symfony\Component\Validator\Constraints\EnumValidator;
+use Symfony\Component\Validator\Exception\ConstraintDefinitionException;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+/**
+ * @author Aurélien Pillevesse <aurelienpillevesse@hotmail.fr>
+ */
+class EnumValidatorTest extends ConstraintValidatorTestCase
```

Comment:
Missing tests:

* Integer-backed enums
* Validating the empty string
* Validating a non-string, e.g. a float, array or object